packages/manager/apps/public-cloud/src/components/easter-eggs/easter-eggs.controller.js

```diff
@@ -0,0 +1,98 @@
+import angular from 'angular';
+import Konami from 'konami-code-js';
+
+import {
+  EASTER_EGGS_LOCAL_STORAGE_KEY,
+  EASTER_EGGS_LOCAL_STORAGE_KEY_ACTIVATE,
+  EASTER_EGGS_LOCAL_STORAGE_KEY_DEACTIVATE,
+  EASTER_EGGS_TRACKING_KEY,
+} from './easter-eggs.constants';
+
+import controller from './modal/controller';
+import template from './modal/template.html';
+
+export default class EeasterEggsController {
+  /* @ngInject */
+  constructor($document, $timeout, $uibModal, atInternet) {
+    this.$document = $document;
+    this.$timeout = $timeout;
+    this.$uibModal = $uibModal;
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.rootElement = this.$document[0].querySelector('html');
+
+    if (EeasterEggsController.isDarkThemeActive()) {
+      this.rootElement.classList.add('theme-dark');
+    }
+
+    this.konami = new Konami(() => this.toggle());
```

Comment:
it may works here because you are not modifying any bound variables in toggle function, but we should keep in mind that in order to trigger an angular digest loop it should be :
```
new Konami(() => $timeout(() => this.toggle()))
```

Reply:
Yep! Here I just applies a class to the root `html` element.